filebeat/values.yaml

```
@@ -135,6 +135,8 @@ affinity: {}
 priorityClassName: ""
 
 updateStrategy: RollingUpdate
+# Only used when updateStrategy is set to "RollingUpdate"
+MaxUnavalaible: 1
```

```suggestion
MaxUnavailable: 1
```

Can you also update the [README](https://github.com/elastic/helm-charts/blob/master/filebeat/README.md#configuration) with this new value?